### PR TITLE
[Method References 2] LPS-168155 WikiImporter 

### PR DESCRIPTION
**Pull request by JanisZhang**
Fixed code conflicts and test
previous: https://github.com/liferay-core-infra/liferay-portal/pull/2566

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:relevant

File: modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/internal/convert/WikiCreoleConvertProcess.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.wiki.web.internal.convert;
+package com.liferay.wiki.internal.convert;
```
Comment (dantewang) [inline modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/internal/convert/WikiCreoleConvertProcess.java:15]: @JanisZhang Why these classes need to be moved together with `MediaWikiImporter`?

File: modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ */
+
+package com.liferay.wiki.constants;
+
+/**
+ * @author Janis Zhang
+ */
+public class WikiImporterKeys {
+
+	public static final String FORMAT_CREOLE = "creole";
```
Comment (dantewang) [inline modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java:22]: @JanisZhang It seems this constant is only used by `MediaWikiImporter` itself.

File: modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+	public static final String OPTIONS_FRONT_PAGE = "OPTIONS_FRONT_PAGE";
+
+	public static final String OPTIONS_IMPORT_LATEST_VERSION =
+		"OPTIONS_IMPORT_LATEST_VERSION";
+
+	public static final String OPTIONS_STRICT_IMPORT_MODE =
+		"OPTIONS_STRICT_IMPORT_MODE";
+
+	public static final String SHARED_IMAGES_CONTENT = "See attachments";
```
Comment (dantewang) [inline modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java:32]: @JanisZhang This one is also only used by `MediaWikiImporter`.

File: modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+package com.liferay.wiki.constants;
+
+/**
+ * @author Janis Zhang
+ */
+public class WikiImporterKeys {
+
+	public static final String FORMAT_CREOLE = "creole";
+
+	public static final String OPTIONS_FRONT_PAGE = "OPTIONS_FRONT_PAGE";
```
Comment (dantewang) [inline modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java:24]: @JanisZhang The `OPTIONS_*` ones are shared constants between Java class and JSP, they should be put in a class named `*WebKeys`.

File: modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+	public static final String OPTIONS_IMPORT_LATEST_VERSION =
+		"OPTIONS_IMPORT_LATEST_VERSION";
+
+	public static final String OPTIONS_STRICT_IMPORT_MODE =
+		"OPTIONS_STRICT_IMPORT_MODE";
+
+	public static final String SHARED_IMAGES_CONTENT = "See attachments";
+
+	public static final String SHARED_IMAGES_TITLE = "SharedImages";
```
Comment (dantewang) [inline modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java:34]: @JanisZhang This one is used by several Java classes, which should be in a class named `*Constants`.

**Comment by catarineaguiar** [comment]
@JanisZhang no regressions found.

File: modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java (hunk 4 above)
Comment (JanisZhang) [reply modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java:24]: got it.

File: modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java (hunk 5 above)
Comment (JanisZhang) [reply modules/apps/wiki/wiki-api/src/main/java/com/liferay/wiki/constants/WikiImporterKeys.java:34]: got it

**Pull request closed without merging**